index.js

```diff
@@ -82,9 +87,108 @@ const sendResponse = ( req, settings, fn ) => {
 		return fn( wpe, null, headers );
 	} );
 
+	if ( processResponseInStreamMode ) {
+		if ( ! req.xhr ) {
+			throw new Error( 'processResponseInStreamMode is not yet implemented for Node.js' );
```

Comment:
Instead of throwing an error, this function is expected to call the callback with the error:
```js
if ( ! req.xhr ) {
  const err = new Error( ... );
  return fn( err, null, headers );
}
```
There's no `body`, but the caller can inspect the `headers` when receiving this error.

Reply:
fixed in 24f855f48bb41fcd1b4393efbd9d4abcb9b9e963